vagrant/boxes.d/99-local.yaml.example

```diff
@@ -15,50 +33,50 @@ centos8-katello-devel:
         - "--foreman-proxy-content-enable-ostree=true"
         - "--katello-devel-modulestream-nodejs=12"
 
-centos8-luna-demo:
-  box: centos8-stream
+almalinux8-luna-demo:
+  box: almalinux8
   memory: 10240
   ansible:
     playbook: playbooks/luna_demo_environment.yml
 
-centos8-hammer-devel:
-  box: centos8-stream
+almalinux8-hammer-devel:
+  box: almalinux8
   memory: 512
   ansible:
     playbook: 'playbooks/hammer_devel.yml'
     group: 'hammer-devel'
     variables: # defaults in roles/hammer_devel/defaults/main.yml
-      #hammer_devel_server: centos8-katello-devel.custom-domain.example.com
+      #hammer_devel_server: almalinux8-katello-devel.custom-domain.example.com
 
 katello-remote-execution:
```

Comment:
I don't think this box makes any sense anymore. katello has a hard dependency on REX now